### DISCUSSION TITLE
on-load MCU::BANK_RIGHT rather than button

### DISCUSSION THREAD

**Discussion by gringow831** [tttapa/Control-Surface#694] [General]
Hi Pieter,

Is there a way of load the MCU::BANK_RIGHT at first rather than using a button to change the bank before I start using the controller?
Then I can use the buttons to go to Bank Left or Right.
So in a way it would be Bank2 Before Bank1... not sure if I explain myself.

Thank you so much

**Comment by gringow831** [comment]
I've tried something like this as the machine takes a few seconds to load I added a delay and after I sent the MCU::Bank_right but it doesn't seem to work always. Not entirely sure if it's me or the software

void setup() {
  RelativeCCSender::setMode(MACKIE_CONTROL_RELATIVE);
  Control_Surface.begin();
  delay(20000);
  Control_Surface.send(MCU::BANK_RIGHT);
}

**Reply by tttapa** [reply]
The `BANK_RIGHT` message in the MCU protocol is a note message, try:

```cpp
Control_Surface.sendNoteOn(MCU::BANK_RIGHT, 127);
```
You might have to send a note off message as well.

**Comment by gringow831** [comment]
Thank you so much for the very quick response! That actually did the trick.

Is there a way to use ManyAddresses alongside with CABLE_x

Bankable::PBPotentiometer<2> knobs[] = {
  {bank, 14, {MCU::VOLUME_1, {MCU::VOLUME_5, CABLE_2}}},

for example I have also tried using CABLENB but over ManyAddresses it could be so much shorter and less variables.
{{bank, BankType::CHANGE_CABLENB}, 14, {MCU::VOLUME_1,}},
